protos/sources/data-node/api/v2/dehistory.proto

```diff
@@ -0,0 +1,119 @@
+syntax = "proto3";
+
+package datanode.api.v2;
+
+import "protoc-gen-openapiv2/options/annotations.proto";
+import "data-node/api/v2/trading_data.proto";
+
+option go_package = "code.vegaprotocol.io/vega/protos/data-node/api/v2";
+
+option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  info: {
+    title: "Vega data node APIs";
+    version: "v0.65.0+dev";
+  }
+  schemes: [HTTP, HTTPS]
+  host: "lb.testnet.vega.xyz/datanode/rest"
+};
+
+service DeHistoryService {
```

Comment:
I don't know how much we really gain security wise having it as a separate service if its assessible on the same port. I thought the plan was to do a local socket admin endpoint like in the core node?
https://github.com/vegaprotocol/vega/tree/develop/core/admin